### ISSUE TITLE
Can you compile from an html string?

### ISSUE THREAD

**Issue by Shak-eah** [wallarm/bundlebars#5]
I misunderstood bundlebars at first: I get that it can compile handlebars templates TO strings, but I wonder if it can also compile hbs templates FROM strings?

Simply speaking,

 bb.compile ('filename.hbs', data) 
vs
bb.compile ( "< h1 >{{username}}< /h1 >", data). 

I need to avoid writing new files and then reading from them, or reading from a file on the disk anyway. I want to be able to use BB's funtionality on a string passed in; is this possible at all?

**Comment by rd5** [comment]
Hey! Thanks for feedback! Can you explain more detailed your use case? Why you can't simply use Handlebars itself to compile strings. e.g.

```
var template = Handlebars.compile('<h1>{{username}}</h1>');
var results = template(data);
```
Which Bundlebars-specific functionality do you want to use here? It looks like a simple to implement feature, but I want to understand purposes of this?

**Comment by Shak-eah** [comment]
Actually, that is exactly what I did!! 

I was looking for something to compile Handlebars since I've been using Ember a lot and hadn't actually messed with the Handlebars portion of it. But after taking a glance at the source code of Bundlebars, I realise that using the Handlebars compiler itself is the best way to approach my specific issue.

I was using Bundlebars because there was a point where I needed to compile files and not strings and - having not looked into Handlebars itself then - had not considered it. The specs of my app changed and I didn't want to let go of Bundlebars as it had made my life easier. 

Thanks very much for the quick response; the purpose, it seems now, is simply to homogenize compilation methods in my app. I would appreciate it greatly if this was implemented as well, but you're right: this can easily be accomplished using Handlebars.
